views/appeal-application/personal-details/postcode-lookup.njk

```diff
@@ -35,7 +33,20 @@
        href="/personal-details/enter-address">{{ i18n.pages.postcodeLookup.cantFind }}</a>
     </br>
     </br>
-    {{ saveButtons('Save and continue', 'Save for later') }}
+    <div class="buttons__container">
+      {{ govukButton({
+        text: i18n.common.buttons.saveAndContinue,
+        name: "buttton",
+        value: "submit",
+        classes: "govuk-!-margin-right-1"
+      }) }}
+
+      {{ govukButton({
+        text: i18n.common.buttons.saveForLater,
+        href: '/task-list',
+        classes: "govuk-button--secondary"
+      }) }}
+    </div>
```

Comment:
Just to be consistent should this be replaced by:
` {{ saveButtons() }}`

then on the `layout.njk` `saveButtons()` we can add the name and value to the save and continue button.